processes.go

```diff
@@ -63,6 +63,14 @@ type Process struct {
 	Memory   constraints.Memory   `json:"Memory,omitempty"`
 	CPUShare constraints.CPUShare `json:"CPUShare,omitempty"`
 	Nproc    constraints.Nproc    `json:"Nproc,omitempty"`
+	Expose   *Exposure            `json:"Expose,omitempty"`
+}
+
+// Exposure holds exposure settings for the Process.
+type Exposure struct {
+	Protocol string `json:"Protocol"`
+	External bool   `json:"External"`
```

Comment:
Yeah. Definitely could see different "visibilities" in the future. I opted to just go with a bool for now, and figure we can just run a migration later if we need to open it up.